front/app/containers/Admin/projects/project/projectHeader/ReviewFlow/index.tsx

```diff
@@ -57,41 +46,33 @@ const ReviewFlow = ({ project }: { project: IProjectData }) => {
   }
 
   const approvalPending = projectReview?.data.attributes.state === 'pending';
-  const approvalGranted = projectReview?.data.attributes.state === 'approved';
-
-  const showProjectApprovalButton = approvalPending && canReview;
-  const showReviewRequestButton =
-    isProjectReviewEnabled && !canReview && !approvalGranted;
-  const showPublishButton =
-    !showProjectApprovalButton && !showReviewRequestButton;
+  const showReviewRequestButton = isProjectReviewEnabled && !canPublish;
+  const showProjectApprovalButton =
+    isProjectReviewEnabled && canPublish && approvalPending;
 
   return (
-    <Box>
-      {showPublishButton && (
-        <Tooltip
-          content={formatMessage(
-            messages.onlyAdminsAndFolderManagersCanPublish,
-            { inFolder: !!project.attributes.folder_id }
-          )}
-          placement="bottom"
-          disabled={canPublish}
+    <Box display="flex" gap="8px">
+      <Tooltip
```

Comment:
The publish button can be showed at all times if we disable it at the right time.